File: vigridr/src/games/guessnumber/sampleclient/ClientLogic.cpp
#include "ClientLogic.h"

#include <stdlib.h>
#include <time.h>

using ::mjollnir::vigridr::Command;
using ::mjollnir::vigridr::WorldModel;
using ::mjollnir::vigridr::GameInit;

Command playTurn(const WorldModel& wm) {
  Command command;
  command.number = rand()%10 + 1;
  return command;
}

void init(const GameInit& gameInit) {
  srand(time(NULL));
}

File: vigridr/src/games/guessnumber/sampleclient/ClientLogic.py
from Command.ttypes import Command
from GameModel.ttypes import GameStatus

from random import randint

class Solution:
    def play_turn(self, wm):
        command = Command(randint(1, 10))
        #print command.number
        return command

    def __init__(self, gameInit):
        pass
